cmd/access-perms.go

```diff
@@ -17,7 +17,12 @@
 
 package cmd
 
-import "path/filepath"
+import (
+	"os"
+
+	json "github.com/minio/colorjson"
+	"github.com/minio/minio-go/v7/pkg/policy"
```

Comment:
No don't use this use this https://pkg.go.dev/github.com/minio/pkg/v2@v2.0.3/policy